plugins/sentinelone/plugin.spec.yaml

```
@@ -2638,7 +2643,7 @@ actions:
         required: true
       type:
         title: Type
-        description: Type of incidents
+        description: Type of incident
         type: string
         enum:
           - threats
```

Is it possible to change this to threat and alert as we're now dealing with a single ID? I realise we'd have to convert it for use in the API back to 'threats' and 'alerts' though.
```suggestion
          - threat
          - alert
        example: threat
```

---

Yes, it's a good point. I changed it already